netty/src/main/java/io/grpc/netty/NettyServer.java

```diff
@@ -81,7 +81,7 @@
   NettyServer(SocketAddress address, Class<? extends ServerChannel> channelType,
               @Nullable EventLoopGroup bossGroup, @Nullable EventLoopGroup workerGroup,
               ProtocolNegotiator protocolNegotiator, int maxStreamsPerConnection,
-              int flowControlWindow, int maxMessageSize, int maxHeaderListSize) {
+      int flowControlWindow, int maxMessageSize, int maxHeaderListSize) {
```

Comment:
My bad, this and the changes in NettyChannelBuilder are left over from adding/removing autoFlowControl to NettyChannelBuilder and NettyServerBuilder. Checking out the originals.